backend-lambda/index.js

```diff
@@ -16,9 +16,9 @@ export const handler = async (event, context) => {
   try {
     if (!pool) {
       const connectionString = process.env.DATABASE_URL;
+      console.log(process.env);
```

Comment:
Let's take this out. It might log out sensitive things.

Reply:
removed in most recent commit (number 6 in this branch)